src/gwt/src/org/rstudio/studio/client/workbench/commands/Commands.cmd.xml

```diff
@@ -427,6 +427,8 @@ well as menu structures (for main menu and popup menus).
          <shortcut refid="jumpTo" value="Shift+Alt+J" if="!org.rstudio.core.client.BrowseCap.isMacintosh()"/>
          <shortcut refid="jumpTo" value="Cmd+Shift+Alt+J" if="org.rstudio.core.client.BrowseCap.isMacintosh()"/>
          <shortcut refid="jumpToMatching" value="Ctrl+P"/>
+         <shortcut refid="selectToMatching" value="Ctrl+Shift+P"/>
+         <shortcut refid="expandToMatching" value="Ctrl+Shift+Alt+P"/>
```

Comment:
Really? Darn, I don't get any special character with that key combination on my Mac (maybe a difference in settings somewhere?)

We can think about a different key combination later.